platformio_tasmota_env32.ini

```diff
@@ -64,6 +64,15 @@ extends                 = env:tasmota32_base
 board                   = esp32_16M
 build_flags             = ${env:tasmota32_base.build_flags} -DFIRMWARE_TASMOTA32
 
+
+[env:tasmota32-sensors]

```

Comment:
There are no plans to introduce a tasmota32-sensors, especially like this one which does nothing differently than tasmota32
As of today Tasmota32 covers the sensors use case
Please remove

Reply:
I removed tasmota32-sensor.